doc/acl/Everflow-test-plan.md

```diff
@@ -77,24 +86,72 @@ everflow_session.json
 ]
 ```
 
-everfow_acl_table.json
+everfow_acl_table.json, ingress, tor->spine
```

Comment:
please do not use this format
test under `roles/test/tasks/everflow` is old and is not used. See `roles/test/tasks/everflow_testbed.yml` and files under `roles/test/tasks/everflow`

Reply:
I have updated the example json files and committed the change.